### PR TITLE
Full website

### PR DESCRIPTION
**Pull request by Christelle-12**
Hello in this project I : 

- Created a website consisting of 3 pages: Home, Calculator, and Quote.
- Used React basic styling like inline styles and imported CSS.
- For every page I created Router 

- All navigation links for all 3 routes are displayed on each page

- Used React router to make all routes work properly

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,12 +1,66 @@
...
+    position: relative;
+    list-style: none;
+  }
+  .current {
+    li {
+      border-bottom: 2px solid black;
+    }
+    
+  }
+`;
```
Comment (KanzaTahreem) [inline src/App.js]: - [ ] Good job with the styles. 👍👍 However, as mentioned in the project requirements, please make sure to use **_React Basic Styling(inline styles, importing CSS and pre-processed CSS files)_** for this milestone 🤓⛏ Following is a quote for a related requirement 👇

  > _In the lesson about styling, we introduced several ways to add styles in a React application. In this project, you should use **React basic styling.**_
  
  - [ ] Also, please make sure to follow the general layout of the wireframe. For example, according to wireframe, on right side, there are links whereas on left side, the App name is present. Similarly, please follow the layout for other pages as well ✨🤓

File: src/index.css
Comment (KanzaTahreem) [inline src/index.css:1]: - [ ] Please make sure to remove the commented code from the codebase to make it look neat and clean 🧹😅

**Review by KanzaTahreem — CHANGES_REQUESTED** [review]
# Changes Required 🔁

Hi @Christelle-12 🙋‍♀️

Good job so far! I really appreciate the effort you put in!  **_However, I did notice a few areas where some additional changes would further enhance the quality of the code. But you are almost there!_** 💪 🏁

so_close

## Highlights 🎯

There are some excellent points that you have made perfectly. Keep up the great work! 👏
- ✅ Correct Gitflow
- ✅ Great job with CSS
- ✅ Descriptive PR and README file
- ✅ Correct Linters

## Required Changes 🛑

> _Please keep in mind that your long hours of work is appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄_

- [ ] I noticed that two sections/headings are present for **_Contributing_** I think it would be better to remove the one highlighted in the image below to make it look more professional. What do you say? 💭📌

image

_Please check the comments under the review._

## Optional suggestions ❕
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [OPTIONAL] Well done for adding future features in the README. I noticed that the first feature is not using the checkout correctly. Please consider adding a space between brackets and text as you did for other features to maintain a consistency🎯

image

 - [OPTIONAL] I think it would be better to add the link to the issues page in the **_Contributing_** section to make it look more professional 🙌


Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by joel-kalema — APPROVED** [review]
Hi @Christelle-12 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Linters are passing 👍 
The project is running successfully 👍 
PR is well descriptive 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@joel-kalema** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**